### DISCUSSION TITLE
Add example/documentation for psql extensions

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#2032] [Feature Requests]
Hi 👋🏻 
There is a lot of psql extensions which are really cool. But when I'm searching some documentations about any function, I don't find any example about how use them.

So my suggestion/idea would be: would it be possible to add some documentations about psql functions please? Like a button which open a popup with some examples kinda like policies or a link to the documentation.

![See the page here: https://user-images.githubusercontent.com/25107942/122801525-86af7200-d2c4-11eb-8c14-d79ac6debe55.png](https://user-images.githubusercontent.com/25107942/122801525-86af7200-d2c4-11eb-8c14-d79ac6debe55.png)

**Comment by awalias** [comment]
nice idea @Olyno I'll make an issue for it internally 👍
